dotnet/src/Extensions/PromptTemplates.Handlebars/Helpers/KernelHelpers/KernelSystemHelpers.cs

```diff
@@ -79,15 +81,48 @@ internal static class KernelSystemHelpers
             variables[name] = value;
         });
 
+        // TODO [@teresaqhoang]: Add tighter restrictions here.
         handlebarsInstance.RegisterHelper("get", (in HelperOptions options, in Context context, in Arguments arguments) =>
         {
+            // Return object or extract specified property from object
```

Comment:
If I understand correctly, arguments should have exactly one or two entries. Can you add some validation of that with a descriptive error if it falls outside those bounds.

Reply:
Yeah, all the system helpers need better argument validation, but for now, it should throw an exception if the expected argument is null 

I'll add it as a fast follow in this work: #3947